.github/workflows/quality_control.yml

```diff
@@ -22,7 +22,10 @@ jobs:
           python-version: 3.11
       - name: "Install all libraries"
         run: |
-          pip3 install -r requirements.txt
+          pip3 install pytest pylint
+          for FILE in $(find . -name 'requirements.txt'); do
+            pip3 install -r $FILE
+          done
           echo "library installation complete"
       - name: "Run Pytest"
         run: |
```

Comment:
Just wondering, will pylint run on all python files, including test files?

Reply:
It will run on all .py files - potential scope later to get it to ignore any files starting with 'test_'